daisy/compute/compute.go

```diff
@@ -39,6 +39,7 @@ type Client interface {
 	DeleteDisk(project, zone, name string) error
 	DeleteImage(project, name string) error
 	DeleteInstance(project, zone, name string) error
+	StopInstance(project, zone, name string) error
```

Comment:
You'll also need to add the test implementation to test_client, thats probably why the tests are failing

Reply:
done, could you check please?